app/controllers/api/user_controller.rb

```diff
@@ -1,7 +1,7 @@
 class Api::UserController < ActionController::Base
   include GDS::SSO::ControllerMethods
 
-  skip_before_filter :verify_authenticity_token
+  skip_before_filter :verify_authenticity_token, raise: false
```

Comment:
In that case, I don't think the commit message matches the problem. My understanding of it now is that if `verify_authenticity_token` isn't defined as a callback and you try to skip it, Rails will raise an error when it evaluates this line.